templates/dart/lib/src/client_base.dart.twig

```diff
@@ -0,0 +1,26 @@
+import 'response.dart';
+import 'client.dart';
+import 'enums.dart';
+
+abstract class ClientBase implements Client {  
+{% for header in spec.global.headers %}
+{% if header.description %}
+    /// {{header.description}}
+{% endif %}
+  ClientBase set{{header.key | caseUcfirst}}(value);
```

Comment:
Or is it necessary that the comment has more indentation?

Reply:
Nope